vendor-bin/doctrine_mongodb/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "autoload": {
```

Comment:
Sure! Got it. I didn't see that the same namespaces were present in the main `composer.json` file. I'll remove those blocks from the children.